src/ampycloud/data.py

```diff
@@ -438,7 +501,92 @@ def metarize(self, which: int = 'slices', base_lvl_alt_perc: float = 5.,
                 boost=self.prms['GROUPING_PRMS']['fluffiness_boost'],
                 **self.prms['LOWESS'])
 
-            # Finally, create the METAR-like code for the cluster
+        return pdf
+
+    def _calculate_sligrolay_base_height(
```

Comment:
As for the function `_calculate_cloud_cover`, how can this be done for the merged groups/layers?

Reply:
It can be done at any time if we have a group/ layer data frame and a data frame holding the original IDs. The non-correspondence between original IDs (the IDs in the data) and the indices in the functions are a source of confusion. But if we want to change this logic, I'd have to think quite a bit, how we could achieve it as we rely on the sorting and reindexing that causes this non-correspondence.